keel-bakery-plugin/src/test/kotlin/com/netflix/spinnaker/keel/bakery/resource/ImageHandlerTests.kt

```diff
@@ -281,7 +261,7 @@ internal class ImageHandlerTests : JUnit5Minutests {
         }
 
         expectThat(request.captured.job.first())
-          .hasEntry("package", "keel_0.160.0-h62.02c0fbf_all.deb")
+          .hasEntry("package", "keel_0.161.0-h63.24d0843_all.deb")
```

Comment:
Why did the version change here?

Reply:
Because I removed `before` block with the mock for `igorService.getArtifact` above, since `upsert` is no longer calling it. That mock had an inconsistency between the version passed in and the package name set in the artifact, which is why this test was passing in the first place. 😃

Reply:
great! Thanks!